File: JAVA/chapter9_examples/chapter9_examples/TestTV.java
package chapter9_examples;

public class TestTV {  
	public static void main(String[] args) {
		// Create tv1
		TV tv1 = new TV();
		tv1.turnOn();
		tv1.setChannel(30);
		tv1.setVolume(3);

		System.out.println("tv1: channel is " + tv1.channel + 
				" and volume level is " + tv1.volumeLevel);

		// Create tv2
		TV tv2 = new TV();
		tv2.turnOn();
		tv2.channelUp();
		tv2.channelUp();
		tv2.volumeUp();

		System.out.println("tv2: channel is " + tv2.channel + 
				" and volume level is " + tv2.volumeLevel);
	}
}

File: Python Basic/TestTV.py
from TV import TV

def main():
    tv1 = TV()
    tv1.turnOn()
    tv1.setChannel(30)
    tv1.setVolume(3)
    
    tv2 = TV()
    tv2.turnOn()
    tv2.channelUp()
    tv2.channelUp()
    tv2.volumeUp()
    
    print("tv1's channel is", tv1.getChannel(), 
        "and volume level is", tv1.getVolumeLevel())
    print("tv2's channel is", tv2.getChannel(),
        "and volume level is", tv2.getVolumeLevel())

main() # Call the main function
